Non-unified source build fix 2025-10 (part 1)

fujii opened the pull request:
#### 9f9f0d22db3f16291e66fccbf0d9ac9847ccca08
<pre>
Non-unified source build fix 2025-10 (part 1)
<a href="https://bugs.webkit.org/show_bug.cgi?id=299936">https://bugs.webkit.org/show_bug.cgi?id=299936</a>

Unreviewed build fix.

* Source/WebCore/Modules/streams/ReadableStream.cpp:
* Source/WebCore/Modules/streams/ReadableStreamBYOBReader.cpp:
* Source/WebCore/animation/BlendingKeyframes.cpp:
* Source/WebCore/animation/KeyframeEffectStack.cpp:
* Source/WebCore/animation/StyleOriginatedAnimation.cpp:
* Source/WebCore/animation/StyleOriginatedAnimationEvent.cpp:
* Source/WebCore/animation/TimelineRangeValue.cpp:
* Source/WebCore/bindings/js/InternalWritableStreamWriter.cpp:
* Source/WebCore/bindings/js/InternalWritableStreamWriter.h:
* Source/WebCore/bindings/js/JSDOMBindingSecurity.cpp:
* Source/WebCore/bindings/js/JSDOMWindowBase.cpp:
* Source/WebCore/bindings/js/JSHTMLElementCustom.cpp:
* Source/WebCore/css/CSSStyleDeclaration.cpp:
* Source/WebCore/css/calc/CSSCalcTree+CalculationValue.cpp:
* Source/WebCore/css/parser/SizesAttributeParser.cpp:
* Source/WebCore/dom/CDATASection.cpp:
* Source/WebCore/dom/ChildListMutationScope.h:
* Source/WebCore/dom/ContentVisibilityDocumentState.cpp:
* Source/WebCore/dom/DataTransfer.cpp:
* Source/WebCore/dom/EventDispatcher.cpp:
* Source/WebCore/dom/EventSender.h:
* Source/WebCore/dom/FindRevealAlgorithms.cpp:
* Source/WebCore/dom/FragmentDirectiveRangeFinder.cpp:
* Source/WebCore/dom/FragmentDirectiveUtilities.cpp:
* Source/WebCore/dom/MutationObserverInterestGroup.h:
* Source/WebCore/dom/MutationObserverRegistration.cpp:
* Source/WebCore/dom/NodeIterator.cpp:
* Source/WebCore/dom/PositionInlines.h:
* Source/WebCore/dom/ProcessingInstruction.cpp:
* Source/WebCore/dom/RadioButtonGroups.cpp:
* Source/WebCore/dom/SerializedNode.cpp:
* Source/WebCore/dom/SimulatedClick.cpp:
* Source/WebCore/editing/AppendNodeCommand.cpp:
* Source/WebCore/editing/ApplyStyleCommand.cpp:
* Source/WebCore/editing/CompositeEditCommand.cpp:
* Source/WebCore/editing/DeleteFromTextNodeCommand.cpp:
* Source/WebCore/editing/DictationCommand.cpp:
* Source/WebCore/editing/EditingStyle.cpp:
* Source/WebCore/editing/Editor.cpp:
* Source/WebCore/editing/FrameSelection.cpp:
* Source/WebCore/editing/InsertIntoTextNodeCommand.cpp:
* Source/WebCore/editing/InsertNodeBeforeCommand.cpp:
* Source/WebCore/editing/MergeIdenticalElementsCommand.cpp:
* Source/WebCore/editing/MoveSelectionCommand.cpp:
* Source/WebCore/editing/RemoveNodeCommand.cpp:
* Source/WebCore/editing/RemoveNodePreservingChildrenCommand.cpp:
* Source/WebCore/editing/ReplaceNodeWithSpanCommand.cpp:
* Source/WebCore/editing/SetNodeAttributeCommand.cpp:
* Source/WebCore/editing/SetSelectionCommand.cpp:
* Source/WebCore/editing/SplitTextNodeCommand.cpp:
* Source/WebCore/editing/TextManipulationController.cpp:
* Source/WebCore/editing/WrapContentsInDummySpanCommand.cpp:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/DOMTokenList.cpp:
* Source/WebCore/html/HTMLAreaElement.cpp:
* Source/WebCore/html/HTMLArticleElement.cpp:
* Source/WebCore/html/HTMLProgressElement.cpp:
* Source/WebCore/html/HTMLStyleElement.cpp:
* Source/WebCore/html/HTMLTableRowElement.cpp:
* Source/WebCore/html/LazyLoadFrameObserver.cpp:
* Source/WebCore/html/LazyLoadImageObserver.cpp:
* Source/WebCore/html/canvas/CanvasStyle.cpp:
* Source/WebCore/html/parser/HTMLDocumentParser.cpp:
* Source/WebCore/html/shadow/DateTimeFieldElement.cpp:
* Source/WebCore/html/shadow/SpinButtonElement.cpp:
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
* Source/WebCore/inspector/DOMEditor.cpp:
* Source/WebCore/inspector/InspectorFrontendHost.cpp:
* Source/WebCore/layout/formattingContexts/grid/PlacedGridItem.h:
* Source/WebCore/layout/formattingContexts/inline/RangeBasedLineBuilder.cpp:
* Source/WebCore/layout/formattingContexts/inline/text/TextUtil.h:
* Source/WebCore/layout/integration/inline/LayoutIntegrationInlineContentBuilder.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationInlineContentPainter.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationLineLayout.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationPagination.cpp:
* Source/WebCore/loader/PolicyChecker.cpp:
* Source/WebCore/mathml/MathMLRowElement.cpp:
* Source/WebCore/page/DOMSelection.cpp:
* Source/WebCore/page/DragController.cpp:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/FocusController.cpp:
* Source/WebCore/page/Frame.cpp:
* Source/WebCore/page/ImageOverlayController.cpp:
* Source/WebCore/page/LargestContentfulPaintData.cpp:
* Source/WebCore/page/LocalFrame.cpp:
* Source/WebCore/page/PointerCaptureController.cpp:
* Source/WebCore/page/PrintContext.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/RemoteFrame.cpp:
* Source/WebCore/page/ScrollBehavior.cpp:
* Source/WebCore/page/SpatialNavigation.h:
* Source/WebCore/page/glib/DragControllerGLib.cpp:
* Source/WebCore/rendering/EllipsisBoxPainter.cpp:
* Source/WebCore/rendering/HitTestResult.cpp:
* Source/WebCore/rendering/RenderBoxModelObject.cpp:
* Source/WebCore/rendering/RenderCounter.cpp:
* Source/WebCore/rendering/RenderDeprecatedFlexibleBox.cpp:
* Source/WebCore/rendering/RenderElement.h:
* Source/WebCore/rendering/RenderGrid.cpp:
* Source/WebCore/rendering/RenderInline.cpp:
* Source/WebCore/rendering/RenderMenuList.cpp:
* Source/WebCore/rendering/RenderSlider.cpp:
* Source/WebCore/rendering/RenderTextControl.cpp:
* Source/WebCore/rendering/RenderVideo.cpp:
* Source/WebCore/rendering/RenderWidget.cpp:
* Source/WebCore/rendering/TextBoxPainter.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceFilterPrimitive.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.cpp:
* Source/WebCore/rendering/svg/SVGTextMetricsBuilder.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceSolidColor.cpp:
* Source/WebCore/style/AttributeChangeInvalidation.cpp:
* Source/WebCore/style/ContainerQueryEvaluator.cpp:
* Source/WebCore/style/MatchedDeclarationsCache.cpp:
* Source/WebCore/style/StyleCustomPropertyRegistry.cpp:
* Source/WebCore/style/values/fonts/StyleFontWidth.cpp:
* Source/WebCore/svg/SVGCursorElement.cpp:
* Source/WebCore/svg/SVGDefsElement.cpp:
* Source/WebCore/svg/SVGFEGaussianBlurElement.cpp:
* Source/WebCore/svg/SVGFETurbulenceElement.cpp:
* Source/WebCore/svg/SVGFitToViewBox.cpp:
* Source/WebCore/svg/SVGGElement.cpp:
* Source/WebCore/svg/SVGGeometryElement.cpp:
* Source/WebCore/svg/SVGMarkerElement.cpp:
* Source/WebCore/svg/SVGPolyElement.cpp:
* Source/WebCore/svg/SVGStopElement.cpp:
* Source/WebCore/svg/SVGSymbolElement.cpp:
* Source/WebCore/svg/properties/SVGAnimatedString.cpp:
* Source/WebCore/xml/XPathExpression.cpp:
* Source/WebCore/xml/XSLStyleSheetLibxslt.cpp:
* Source/WebCore/xml/XSLTProcessor.cpp:
* Source/WebCore/xml/XSLTProcessorLibxslt.cpp:

Canonical link: <a href="https://commits.webkit.org/300828@main">https://commits.webkit.org/300828@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9b8af370202fac4c5a65d874d96efc927ec81d55

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/123993 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/43708 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/34405 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/130816 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/76145 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/125870 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/44432 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/52302 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/130816 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/62578 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/126947 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/132/builds/35407 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/110919 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/130816 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/34356 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/29084 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/74296 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/105138 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/29303 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/133489 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/50946 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/38808 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/102778 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/51320 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/107139 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/102600 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 vision-wk2~~](https://ews-build.webkit.org/#/builders/126/builds/47957 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/26207 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/47813 "Failed to checkout and rebase branch from PR 51621") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19494 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/50799 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/56563 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/50273 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/53619 "Built successfully") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/51947 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/34ebef670e159269436b512cc2b0ab8993a84336. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51621

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/9b8af370202fac4c5a65d874d96efc927ec81d55. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51621

fujii committed:
Non-unified source build fix 2025-10 (part 1)
https://bugs.webkit.org/show_bug.cgi?id=299936

Unreviewed build fix.

* Source/WebCore/Modules/streams/ReadableStream.cpp:
* Source/WebCore/Modules/streams/ReadableStreamBYOBReader.cpp:
* Source/WebCore/animation/BlendingKeyframes.cpp:
* Source/WebCore/animation/KeyframeEffectStack.cpp:
* Source/WebCore/animation/StyleOriginatedAnimation.cpp:
* Source/WebCore/animation/StyleOriginatedAnimationEvent.cpp:
* Source/WebCore/animation/TimelineRangeValue.cpp:
* Source/WebCore/bindings/js/InternalWritableStreamWriter.cpp:
* Source/WebCore/bindings/js/InternalWritableStreamWriter.h:
* Source/WebCore/bindings/js/JSDOMBindingSecurity.cpp:
* Source/WebCore/bindings/js/JSDOMWindowBase.cpp:
* Source/WebCore/bindings/js/JSHTMLElementCustom.cpp:
* Source/WebCore/css/CSSStyleDeclaration.cpp:
* Source/WebCore/css/calc/CSSCalcTree+CalculationValue.cpp:
* Source/WebCore/css/parser/SizesAttributeParser.cpp:
* Source/WebCore/dom/CDATASection.cpp:
* Source/WebCore/dom/ChildListMutationScope.h:
* Source/WebCore/dom/ContentVisibilityDocumentState.cpp:
* Source/WebCore/dom/DataTransfer.cpp:
* Source/WebCore/dom/EventDispatcher.cpp:
* Source/WebCore/dom/EventSender.h:
* Source/WebCore/dom/FindRevealAlgorithms.cpp:
* Source/WebCore/dom/FragmentDirectiveRangeFinder.cpp:
* Source/WebCore/dom/FragmentDirectiveUtilities.cpp:
* Source/WebCore/dom/MutationObserverInterestGroup.h:
* Source/WebCore/dom/MutationObserverRegistration.cpp:
* Source/WebCore/dom/NodeIterator.cpp:
* Source/WebCore/dom/PositionInlines.h:
* Source/WebCore/dom/ProcessingInstruction.cpp:
* Source/WebCore/dom/RadioButtonGroups.cpp:
* Source/WebCore/dom/SerializedNode.cpp:
* Source/WebCore/dom/SimulatedClick.cpp:
* Source/WebCore/editing/AppendNodeCommand.cpp:
* Source/WebCore/editing/ApplyStyleCommand.cpp:
* Source/WebCore/editing/CompositeEditCommand.cpp:
* Source/WebCore/editing/DeleteFromTextNodeCommand.cpp:
* Source/WebCore/editing/DictationCommand.cpp:
* Source/WebCore/editing/EditingStyle.cpp:
* Source/WebCore/editing/Editor.cpp:
* Source/WebCore/editing/FrameSelection.cpp:
* Source/WebCore/editing/InsertIntoTextNodeCommand.cpp:
* Source/WebCore/editing/InsertNodeBeforeCommand.cpp:
* Source/WebCore/editing/MergeIdenticalElementsCommand.cpp:
* Source/WebCore/editing/MoveSelectionCommand.cpp:
* Source/WebCore/editing/RemoveNodeCommand.cpp:
* Source/WebCore/editing/RemoveNodePreservingChildrenCommand.cpp:
* Source/WebCore/editing/ReplaceNodeWithSpanCommand.cpp:
* Source/WebCore/editing/SetNodeAttributeCommand.cpp:
* Source/WebCore/editing/SetSelectionCommand.cpp:
* Source/WebCore/editing/SplitTextNodeCommand.cpp:
* Source/WebCore/editing/TextManipulationController.cpp:
* Source/WebCore/editing/WrapContentsInDummySpanCommand.cpp:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/DOMTokenList.cpp:
* Source/WebCore/html/HTMLAreaElement.cpp:
* Source/WebCore/html/HTMLArticleElement.cpp:
* Source/WebCore/html/HTMLProgressElement.cpp:
* Source/WebCore/html/HTMLStyleElement.cpp:
* Source/WebCore/html/HTMLTableRowElement.cpp:
* Source/WebCore/html/LazyLoadFrameObserver.cpp:
* Source/WebCore/html/LazyLoadImageObserver.cpp:
* Source/WebCore/html/canvas/CanvasStyle.cpp:
* Source/WebCore/html/parser/HTMLDocumentParser.cpp:
* Source/WebCore/html/shadow/DateTimeFieldElement.cpp:
* Source/WebCore/html/shadow/SpinButtonElement.cpp:
* Source/WebCore/html/shadow/TextControlInnerElements.cpp:
* Source/WebCore/inspector/DOMEditor.cpp:
* Source/WebCore/inspector/InspectorFrontendHost.cpp:
* Source/WebCore/layout/formattingContexts/grid/PlacedGridItem.h:
* Source/WebCore/layout/formattingContexts/inline/RangeBasedLineBuilder.cpp:
* Source/WebCore/layout/formattingContexts/inline/text/TextUtil.h:
* Source/WebCore/layout/integration/inline/LayoutIntegrationInlineContentBuilder.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationInlineContentPainter.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationLineLayout.cpp:
* Source/WebCore/layout/integration/inline/LayoutIntegrationPagination.cpp:
* Source/WebCore/loader/PolicyChecker.cpp:
* Source/WebCore/mathml/MathMLRowElement.cpp:
* Source/WebCore/page/DOMSelection.cpp:
* Source/WebCore/page/DragController.cpp:
* Source/WebCore/page/EventHandler.cpp:
* Source/WebCore/page/FocusController.cpp:
* Source/WebCore/page/Frame.cpp:
* Source/WebCore/page/ImageOverlayController.cpp:
* Source/WebCore/page/LargestContentfulPaintData.cpp:
* Source/WebCore/page/LocalFrame.cpp:
* Source/WebCore/page/PointerCaptureController.cpp:
* Source/WebCore/page/PrintContext.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/RemoteFrame.cpp:
* Source/WebCore/page/ScrollBehavior.cpp:
* Source/WebCore/page/SpatialNavigation.h:
* Source/WebCore/page/glib/DragControllerGLib.cpp:
* Source/WebCore/rendering/EllipsisBoxPainter.cpp:
* Source/WebCore/rendering/HitTestResult.cpp:
* Source/WebCore/rendering/RenderBoxModelObject.cpp:
* Source/WebCore/rendering/RenderCounter.cpp:
* Source/WebCore/rendering/RenderDeprecatedFlexibleBox.cpp:
* Source/WebCore/rendering/RenderElement.h:
* Source/WebCore/rendering/RenderGrid.cpp:
* Source/WebCore/rendering/RenderInline.cpp:
* Source/WebCore/rendering/RenderMenuList.cpp:
* Source/WebCore/rendering/RenderSlider.cpp:
* Source/WebCore/rendering/RenderTextControl.cpp:
* Source/WebCore/rendering/RenderVideo.cpp:
* Source/WebCore/rendering/RenderWidget.cpp:
* Source/WebCore/rendering/TextBoxPainter.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceFilterPrimitive.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceGradient.cpp:
* Source/WebCore/rendering/svg/RenderSVGViewportContainer.cpp:
* Source/WebCore/rendering/svg/SVGTextMetricsBuilder.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceSolidColor.cpp:
* Source/WebCore/style/AttributeChangeInvalidation.cpp:
* Source/WebCore/style/ContainerQueryEvaluator.cpp:
* Source/WebCore/style/MatchedDeclarationsCache.cpp:
* Source/WebCore/style/StyleCustomPropertyRegistry.cpp:
* Source/WebCore/style/values/fonts/StyleFontWidth.cpp:
* Source/WebCore/svg/SVGCursorElement.cpp:
* Source/WebCore/svg/SVGDefsElement.cpp:
* Source/WebCore/svg/SVGFEGaussianBlurElement.cpp:
* Source/WebCore/svg/SVGFETurbulenceElement.cpp:
* Source/WebCore/svg/SVGFitToViewBox.cpp:
* Source/WebCore/svg/SVGGElement.cpp:
* Source/WebCore/svg/SVGGeometryElement.cpp:
* Source/WebCore/svg/SVGMarkerElement.cpp:
* Source/WebCore/svg/SVGPolyElement.cpp:
* Source/WebCore/svg/SVGStopElement.cpp:
* Source/WebCore/svg/SVGSymbolElement.cpp:
* Source/WebCore/svg/properties/SVGAnimatedString.cpp:
* Source/WebCore/xml/XPathExpression.cpp:
* Source/WebCore/xml/XSLStyleSheetLibxslt.cpp:
* Source/WebCore/xml/XSLTProcessor.cpp:
* Source/WebCore/xml/XSLTProcessorLibxslt.cpp:

Canonical link: https://commits.webkit.org/300828@main

webkit-commit-queue commented:
Committed 300828@main (9f9f0d22db3f): <https://commits.webkit.org/300828@main>

Reviewed commits have been landed. Closing PR #51621 and removing active labels.
